pkg/auth/server_config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2022 sealos.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/labring/sealos/pkg/auth/tools"
+	"k8s.io/client-go/kubernetes"
+)
+
+var clientset *kubernetes.Clientset
+
+func ConfigRegister(auth *restful.WebService)  {
+	auth.Path("/").
+		Consumes("*/*").
+		Produces(restful.MIME_JSON) // you can specify this per route as well
+	//user token http
+	auth.Route(auth.GET("/login").To(handlerLogin))
```

Comment:
add some discribe for those two handler, /login for callback, and /config for got the user kubeconfig?